spec/system/course_lesson_completion_spec.rb

```diff
@@ -34,7 +34,9 @@
     it 'cannot complete a lesson' do
       visit path_course_path(path, course)
 
-      expect(page).not_to have_css('[data-test_id="complete-button"]')
+      expect(page).to have_css('[data-test-id]')
+
+      expect(page).to have_no_css('[data-test-id="complete-button"]')
```

Comment:
`data-test_id` was a subtle typo (or it's changed at some point), one of the dangers of using negative selectors.

Fixed while I was here for the new Rubocop offence